### PR TITLE
Fix NullPointerException for testCustomCodeRunner.

### LINKED ISSUE #572: NPE in test testCustomCodeRunner
2999706 [HelixController-async_tasks-CLUSTER_TestHelixCustomCodeRunner] ERROR org.apache.helix.controller.stages.BestPossibleStateCalcStage  - Event b7e431cd_DEFAULT :
 Could not update cluster status metrics!
java.lang.NullPointerException
        at org.apache.helix.monitoring.mbeans.ClusterStatusMonitor.setPerInstanceResourceStatus(ClusterStatusMonitor.java:416)
        at org.apache.helix.controller.stages.BestPossibleStateCalcStage$1.call(BestPossibleStateCalcStage.java:89)
        at java.util.concurrent.FutureTask.run(FutureTask.java:266)
        at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.access$201(ScheduledThreadPoolExecutor.java:180)
        at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.run(ScheduledThreadPoolExecutor.java:293)
        at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142)
        at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617)
        at java.lang.Thread.run(Thread.java:745)
2999714 [HelixController-async_tasks-CLUSTER_TestHelixCustomCodeRunner] ERROR org.apache.helix.controller.stages.BestPossibleStateCalcStage  - Event 6e898242_DEFAULT :
 Could not update cluster status metrics!
java.lang.NullPointerException
        at org.apache.helix.monitoring.mbeans.ClusterStatusMonitor.setPerInstanceResourceStatus(ClusterStatusMonitor.java:416)
        at org.apache.helix.controller.stages.BestPossibleStateCalcStage$1.call(BestPossibleStateCalcStage.java:89)
        at java.util.concurrent.FutureTask.run(FutureTask.java:266)
        at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.access$201(ScheduledThreadPoolExecutor.java:180)
        at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.run(ScheduledThreadPoolExecutor.java:293)
        at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142)
        at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617)
        at java.lang.Thread.run(Thread.java:745)

### PR DESCRIPTION
**Pull request by pkuwm**
### Issues

- [x] My PR addresses the following Helix issues and references them in the PR description:

Fixes #572 

### Description

- [x] Here are some details about my PR, including screenshots of any UI changes:

The new live instance doesn't have its instance config. This causes NullPointerException when monitor is trying to get the instance config.
Instance config is needed for the new live instance.

### Tests

- [ ] The following tests are written for this issue:

(List the names of added unit/integration tests)

- [x] The following is the result of the "mvn test" command on the appropriate module:

[INFO] Results:
[INFO]
[ERROR] Failures:
[ERROR]   TestAlertingRebalancerFailure.testTagSetIncorrect:174->checkResourceBestPossibleCalFailureState:310 expected:<true> but was:<false>
[ERROR]   TestTaskPerformanceMetrics.testTaskPerformanceMetrics:118 expected:<true> but was:<false>
[INFO]
[ERROR] Tests run: 882, Failures: 2, Errors: 0, Skipped: 1
[INFO]
[INFO] ------------------------------------------------------------------------
[INFO] BUILD FAILURE
[INFO] ------------------------------------------------------------------------
[INFO] Total time:  57:22 min
[INFO] Finished at: 2019-11-14T18:35:23-08:00
### Commits

- [x] My commits all reference appropriate Apache Helix GitHub issues in their subject lines, and I have squashed multiple commits if they address the same issue. In addition, my commits follow the guidelines from "[How to write a good git commit message](http://chris.beams.io/posts/git-commit/)":
  1. Subject is separated from body by a blank line
  1. Subject is limited to 50 characters (not including Jira issue reference)
  1. Subject does not end with a period
  1. Subject uses the imperative mood ("add", not "adding")
  1. Body wraps at 72 characters
  1. Body explains "what" and "why", not "how"

### Documentation

- [x] In case of new functionality, my PR adds documentation in the following wiki page:

(Link the GitHub wiki you added)

### Code Quality

- [x] My diff has been formatted using helix-style.xml

### CODE REVIEW

**Comment by narendly** [comment]
The new live instance doesn't its instance config. -> fix description please.

File: helix-core/src/test/java/org/apache/helix/integration/TestHelixCustomCodeRunner.java [hunk 1]
```diff
@@ -127,7 +123,10 @@ public void testCustomCodeRunner() throws Exception {
     for (int i = 0; i < nodeNb; i++) {
       participants[i].syncStop();
     }
-    accessor.removeProperty(keyBuilder.liveInstance("newLiveInstance"));
+    HelixDataAccessor accessor =
+        new ZKHelixDataAccessor(_clusterName, new ZkBaseDataAccessor<>(_gZkClient));
+    Builder keyBuilder = accessor.keyBuilder();
+    accessor.removeProperty(keyBuilder.liveInstance("localhost_1000"));
```
Comment (narendly) [inline helix-core/src/test/java/org/apache/helix/integration/TestHelixCustomCodeRunner.java]: Nit: make sure instanceconfigs are deleted at the end of the test. Probably dropCluster() takes care of them? Let's confirm.

Comment (pkuwm) [reply helix-core/src/test/java/org/apache/helix/integration/TestHelixCustomCodeRunner.java]: Yes, instance configs are deleted in dropCluster().

**Comment by pkuwm** [comment]
This PR is ready to be merged, approved by @jiajunwang  @narendly
Commit:
```
The new live instance doesn't have its instance config. This causes NullPointerException when monitor is trying to get the instance config. 
Changelist:
Add instance config for the new live instance.
```

**Pull request merged**